doc/src/docs/InputOutputReference/01a-InputOutputReference.md

```diff
@@ -4412,119 +4412,179 @@ Material:AirGap,B1 - AIRSPACE RESISTANCE,  ! Material Name
 
+$$ d_{EMPD,deep} = \sqrt{\frac{\delta_{perm} P_{sat} \tau_{deep}} {\rho_{material} \frac{du}{d\phi} \pi}} $$
 
- MaterialProperty:MoisturePenetrationDepth:Settings,
-    E1 - 3 / 4 IN PLASTER OR GYP BOARD,  !- Name
-    0.004,                   !- Effective Moisture Penetration Depth {m}
-    0.072549,           !- Moisture Equation Coefficient a {dimensionless}
-    0.397173,           !- Moisture Equation Coefficient b {dimensionless}
-    0.007774,           !- Moisture Equation Coefficient c {dimensionless}
-    11.7057;            !- Moisture Equation Coefficient d {dimensionless}
-
-
-  Material,
-    C10 - 8 IN HW CONCRETE,  !- Name
-    MediumRough,             !- Roughness
-    0.2033016,               !- Thickness {m}
-    1.729577,                !- Conductivity {W/m-K}
-    2242.585,                !- Density {kg/m3}
-    836.8000,                !- Specific Heat {J/kg-K}
-    0.9000000,               !- Thermal Absorptance
-    0.6500000,               !- Solar Absorptance
-    0.6500000;               !- Visible Absorptance
+where each term is the same as the surface layer, except that the cycle period is different. This is usually on the order of weeks for the deep layer.
+
+#### Field: Coating Layer Thickness
+The coating layer thickness (in meters) adds an additional resistance between the surface layer and the zone and represents a thin coating, such as paint, plaster, or other wall coverings.
+
+This input is optional, and an input of zero implies no coating.
+
+#### Field: Coating Layer Water Vapor Diffusion Resistance Factor
 
+The vapor diffusion resistance factor of the coating is the coating's resistance to water vapor diffusion *relative* to the resistance to water vapor diffusion in stagnant air (see Vapor diffusion resistance factor section above).
 
- MaterialProperty:MoisturePenetrationDepth:Settings,
-    C10 - 8 IN HW CONCRETE,  !- Name
-    0.004,                   !- Effective Moisture Penetration Depth {m}
-    0.018062,             !- Moisture Equation Coefficient a {dimensionless}
-    0.451879,             !- Moisture Equation Coefficient b {dimensionless}
-    0.026178,             !- Moisture Equation Coefficient c {dimensionless}
-    10.8356;              !- Moisture Equation Coefficient d {dimensionless}
+This input is optional, and an input of zero implies no coating.
+
+Below are two IDF examples:
+
+This set of inputs can be used for aerated concrete (assuming linear sorption curve). Density, input elsewhere, is :
+
+```idf
+MaterialProperty:MoisturePenetrationDepth:Settings,
```

Comment:
And the idf snippet field names in the I/O Ref.  And one other nit-pick, the output variable names below should be title case (the SetupOutput calls have it correct).